security/nbde-implementation-guide.adoc

```diff
@@ -0,0 +1,75 @@
+// CNF-2127 assembly
+[id="nbde-implementation-guide"]
+= Implementing Network-Bound Disk Encryption (NBDE)
```

Comment:
Stephen, in the [mod docs guidelines](https://redhat-documentation.github.io/modular-docs/#what-modular-documentation-is), there is the suggestion that each assembly documents a user story.

I'm sure there's some flexibility in that guideline, but this assembly pushes it too far.  Several user stories are covered in this assembly.  To meet the mod docs guidelines, break this into several assemblies and look at the "File Integrity Operator" section of the topic map as an example for how to group the assemblies under a TOC node like "Network-Bound Disk Encryption (NBDE)".  You have enough information to justify it.

Reply:
I had an afterthought on this comment.  I failed to mention to use sentence case for this title and in the _topic_map.yml file. This is covered in the [Assembly/Module titles and section headings](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings) section of the doc guidelines.

Reply:
@mikemckiernan 
I'll have a look at the guidelines, but please know that this was developed under just one user story - TELCODOCS-266. Is it a really large story?  Yup.

Reply:
@mikemckiernan I've restructured this section. Plz review and let me know.